### PR TITLE
French localization: translation of empty strings

### PR DESCRIPTION
**Pull request by mlep**
- [ ] Change in CHANGELOG.md described?
- [ ] Changes in pull request outlined? (What, why, ...)
- [ ] Tests created for changes?
- [ ] Tests green?

### CODE REVIEW

File: src/main/resources/l10n/JabRef_fr.properties [hunk 1]
```diff
@@ -1680,57 +1680,57 @@ New_%0_database=Nouvelle_base_%0
...
+link_should_refer_to_a_correct_file_path=Le_lien_doit_correspondre_à_un_chemin_de_fichier_valide
+
+abbreviation_detected=Abréviation_détectée
+wrong_entry_type_as_proceedings_has_page_numbers=Mauvais_type_d'entrée_car_le_proceedings_a_des_numéros_de_page
+Copy_special=Copie_spéciale
+
+Abbreviate_journal_names=Abrégez_les_noms_de_journaux
+Abbreviating...=Abrègement_en_cours
+Adding_fetched_entries=Ajout_des_entrées_récupérées
+Are_you_sure_you_want_to_remove_the_already_existent_SQL_DBs?=Etes-vous_sûr_de_vouloir_supprimer_une_base_SQL_existante_?
```
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_fr.properties]: There should not be a whitespace in front of the "?", or?

File: src/main/resources/l10n/JabRef_fr.properties [hunk 2]
```diff
@@ -1680,57 +1680,57 @@ New_%0_database=Nouvelle_base_%0
...
+Display_keywords_appearing_in_ANY_entry=Afficher_les_mots-clefs_apparaissant_dans_AU_MOINS_UNE_entrée
+Fetching_entries_from_Inspire=Recherche_d'entrées_depuis_Inspire
+Multiple_entries=Entrées_multiples
+None_of_the_selected_entries_have_BibTeX_keys.=Aucune_des_entrées_sélectionnées_n'a_une_clef_BibTeX
+Resolve_duplicate_keys=Traite_les_clefs_dupliquées
+Unabbreviate_journal_names=Dés-abréger_des_noms_de_journaux
+Unabbreviating...=Dés-abrègement...
+Usage=Usage
+
+Are_you_sure_you_want_to_reset_all_settings_to_default_values?=Etes-vous_sûr_de_vouloir_rétablir_toutes_les_valeurs_par_défaut_du_paramétrage_?
```
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_fr.properties]: Whitespace " ?"=

File: src/main/resources/l10n/JabRef_fr.properties [hunk 3]
```diff
@@ -1680,57 +1680,57 @@ New_%0_database=Nouvelle_base_%0
...
+Reset_preferences=Rétablir_les-préférences
+
+Ill-formed_entrytype_comment_in_bib_file=Commentaire_de_type_d'entrées_mal_construit_dans_un_fichier_bib
+Clipboard=Presse-papiers
+Could_not_paste_entry_as_text\:=L'entrée_n'a_pas_pu_être_collée_comme_du_texte_:
```
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_fr.properties]: Whitespace: "... :" ?

File: src/main/resources/l10n/JabRef_fr.properties [hunk 4]
```diff
@@ -1680,57 +1680,57 @@ New_%0_database=Nouvelle_base_%0
...
+Do_you_still_want_to_continue?=Voulez-vous_continuer_?
```
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_fr.properties]: Whitespace?

File: src/main/resources/l10n/JabRef_fr.properties [hunk 5]
```diff
@@ -1680,57 +1680,57 @@ New_%0_database=Nouvelle_base_%0
...
+Please_enter_the_desired_name\:=Entrez,_s'il_vous_plaît,_le_nom_souhaité_:
```
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_fr.properties]: Whitespace?

File: src/main/resources/l10n/JabRef_fr.properties [hunk 6]
```diff
@@ -1680,57 +1680,57 @@ New_%0_database=Nouvelle_base_%0
...
+SQL_Export=Exportation_SQL
+This_action_will_modify_the_following_field(s)_in_at_least_one_entry_each\:=Cette_action_modifiera_le(s)_champ(s)_suivant(s)_dans_au_moins_une_entrée_chacune_:
```
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_fr.properties]: Whitespace?

File: src/main/resources/l10n/JabRef_fr.properties (hunk 1 above)
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_fr.properties]: Okay I just found out, that you french guys place a whitespace in front of question marks ;-)
So forget about my comments...

**Comment by mlep** [comment]
@matthiasgeiger : Hello, could this pull request be merged?

**Comment by oscargus** [comment]
@mlep: can you please rebase and I'll merge it.

**Comment by mlep** [comment]
Hello @oscargus , I do not what does "rebase" exactly means...  In fact, I carried out all these changes online.

**Comment by oscargus** [comment]
Ah, OK! It basically means "apply your changes again on the latest version
of the file", which is done by a command so no need to actually do it again
for you.  No need to worry though. I'm sure someone with more knowledge
than me can merge this. (The reason being that someone managed to squeeze
in a few new strings, so now Github can not simply combine the files.)

**Comment by simonharrer** [comment]
Hm, I just merge this in by following the instructions given at the link under **command line instructions**.

**Comment by oscargus** [comment]
Oh there's a link. :-)

**Comment by mlep** [comment]
Thanks for merging.
Where is the link ?!?

**Comment by simonharrer** [comment]
The link is no longer here, as it only appears when there is a conflict with the pull request. :)

**Comment by mlep** [comment]
Was it something like:
> This branch has conflicts that must be resolved
> Only those with write access to this repository can merge pull requests.

?

**Comment by simonharrer** [comment]
Yes, but as I do have the rights, it shows a link there with instructions to merge it manually.

**Comment by mlep** [comment]
OK. I first thought I had missed to read the instructions...

**Pull request closed without merging**